plugins/httpjson/httpjson.go

```diff
@@ -126,7 +133,7 @@ func (h *HttpJson) gatherServer(acc plugins.Accumulator, service Service, server
 		return err
 	}
 
-	var jsonOut interface{}
+	var jsonOut map[string]interface{}
```

Comment:
@palkan supporting floats could become very complicated, there tends to be a lot of metadata in JSON that users probably don't want. Also storing strings as InfluxDB values has bad compression and isn't very useful.